lib/artsy-eventservice/artsy/event_service.rb

```diff
@@ -2,6 +2,23 @@
 # frozen_string_literal: true
 module Artsy
   module EventService
+
+    # Post data without an event, must be serialized.
```

Comment:
"Serialized" how?

Reply:
i.e. as JSON

Reply:
RabbitMQ is serialization-agnostic.  You can even send Ruby-Marshalled objects through it (tho this is not a great idea)